app/code/Bulbulatory/Recomendations/Controller/Recommendations/ConfirmRecommendation.php

```diff
@@ -0,0 +1,48 @@
+<?php
+namespace Bulbulatory\Recomendations\Controller\Recommendations;
+
+use Bulbulatory\Recomendations\Api\Data\RecommendationInterface;
+use Bulbulatory\Recomendations\Api\RecommendationRepositoryInterface;
+use Magento\Framework\Controller\ResultFactory;
+use Magento\Customer\Model\Session;
+use Magento\Backend\App\Action\Context;
+
+class ConfirmRecommendation extends \Magento\Framework\App\Action\Action
+{
+    protected $recommendation;
```

Comment:
Nigdzie nie widzę, żeby było to wykorzystane, więc chyba można skasować tę deklarację oraz wpisy w konstruktorze.